apps/scan/backend/src/interpret.test.ts

```diff
@@ -34,6 +42,36 @@ afterEach(async () => {
   await fs.rm(ballotImagesPath, { recursive: true });
 });
 
+test('configure get/set', () => {
```

Comment:
I added these tests because I had previously factored out `scan/backend/src/interpret.ts` to `libs/ballot-interpreter`. That dropped coverage in the latter package so I brought coverage back up with these tests. I reverted my changes that factored out the package, but thought I'd leave the tests in for coverage anyway.